Gems/MiniAudio/Code/Include/MiniAudio/MiniAudioListenerBus.h

```diff
@@ -23,6 +23,10 @@ namespace MiniAudio
         virtual AZ::u32 GetChannelCount() const = 0;
 
         //! Cone controls for directional attenuation
```

Comment:
nit: should this comment be updated?

Reply:
Fixed, see: b0c890bee3c6a496dd79da8fdd1f2add0e304d64